cdap-docs/introduction/source/index.rst

```diff
@@ -0,0 +1,1131 @@
+
+.. list-table::
+   :widths: 45 45 10
+   :header-rows: 1
+
+   * - New Paradigm With CDAP
+     - Current Approach and Required Technologies
+     - 
+     
+   * - ``$ get endpoints service Wise.WiseService``
+     - - Navigate to the resouce manager UI
+       - Find the Wise.WiseService on UI
+       - Click to the see application logs
+       - Find all the node managers for the application containers
+       - Navigate to all the containers in sepearate tabs 
+       - Click on container logs
+     - - HDFS
+       - Kafka
+       - Hive
+       - Oozie
+       - YARN
+   
+   * - ``$ call service Wise.WiseService GET /ip/69.181.160.120/count``
+     - - Discover the host and port where the service is running on by looking at the host 
+         and port in the YARN logs or by writing a discovery client that is co-ordinated using Zookeeper
+       - Run ``curl http://hostname:port/ip/69.181.160.120/count``
+     - - HDFS
+       - Kafka
+       - Hive
+       - Oozie
+       - YARN
+   
+   * - ``$ list dataset instances``
+         - ``cdap.user.bounceCountStore``
+         - ``cdap.user.pageViewStore``
+     - - Run the following command in Hbase shell
+       - ``hbase shell> list "cdap.user.*"``
+     - - HDFS
+       - Kafka
+       - Hive
+       - Oozie
+       - YARN
+       - HBase
+
+.. rubric:: View bounce count results 
+
+.. list-table::
+   :widths: 45 45 10
+   :header-rows: 1
+
+   * - New Paradigm With CDAP
+     - Current Approach and Required Technologies
+     - 
+     
+   * - ``$ execute 'SELECT * FROM user_bouncecountstore LIMIT 5'``
+     - - Run the folllowing command in Hive CLI
+       - ``"SELECT * FROM user_bouncecountstore LIMIT 5"``
+     - - HDFS
+       - Kafka
+       - Hive
+       - Oozie
+       - YARN
+       - HBase
+   
+   * - ``$ stop service Wise.WiseService``
+     - - Find the yarn application Id from the following command
+       - ``yarn application -list | grep "Wise.WiseService"``
+       - Stop the application by running the following command
+       - ``yarn application -kill <Application ID>``
+     - - HDFS
+       - Kafka
+       - Hive
+       - Oozie
+       - YARN
+       - HBase
+   
+   * - ``$ stop flow Wise.WiseFlow``
+     - - Find the yarn application Id from the following command
+       - ``yarn application -list | grep "Wise.WiseFlow"``
+       - Stop the application by running the following command
+       - ``yarn application -kill <Application ID>``
+     - - HDFS
+       - Kafka
+       - Hive
+       - Oozie
+       - YARN
+       - HBase
+  
+   * - ``$ delete app Wise``
+     - - Delete the workflow from oozie
+       - Remove the service jars and flow jars
+     - - HDFS
+       - Kafka
+       - Hive
+       - Oozie
+       - YARN
+       - HBase
+
+Summary
+=======
+
+.. list-table::
+   :widths: 45 45 10
+   :header-rows: 1
+
+   * - New Paradigm With CDAP
+     - Current Approach and Required Technologies
+     - 
+
+   * - CDAP
```

Comment:
Probably want to list corresponding points in CDAP here. For example,

CDAP: Learn a single framework that works with multiple technologies
Without CDAP: Familiarize and learn how to operationalize the different technologies